docs/Sonarr/Sonarr-Release-Profile-RegEx.md

```diff
@@ -430,8 +430,7 @@ Add this to your `Preferred (3)` with a score of [15]
 Add this to your `Must not contain (2)`
 
 ```bash
-/\b(dv|dovi|dolby[ .]vision)\b.*(?=[ ._-]web[ ._-]?(dl|rip)\b)/i,
-/(?=[ ._-]web[ ._-]?(dl|rip)\b).*\b(dv|dovi|dolby[ .]vision)\b/i
+\b(?=.*(dv|dovi|dolby[ .]vision)\b)(?=.*(WEB[-_. ]DL|WEBDL|AmazonHD|iTunesHD|MaxdomeHD|NetflixU?HD|WebHD|[. ]WEB[. ](?:[xh]26[45]|DDP?5[. ]1)|[. ](?-i:WEB)$|\d+0p(?:[-. ]AMZN)?[-. ]WEB[-. ]|WEB-DLMux|\b\s\/\sWEB\s\/\s\b|AMZN[. ]WEB[. ]|WebRip|Web-Rip|WEBMux))
```

Comment:
```suggestion
/(?:\b(?:.*(dv|dovi|dolby[ .]vision)\b)(?:.*(?:WEB[-_. ]?DL|AmazonHD|iTunesHD|MaxdomeHD|NetflixU?HD|WebHD|[. ]WEB[. ](?:[xh][.]?26[45]|DDP?5[. ]1)|WEB-DLMux|\b\s\/\sWEB\s\/\s\b|AMZN[. -]WEB[. -]|Web-?Rip|WEBMux)))/i
```

- cleans up some duplicates
- ensures it stays case insensitive
- captures only the various DV as a group